initialinstructions.md

```diff
@@ -0,0 +1,28 @@
+---
+title: InitialInstructions
+filename: initialinstructions.md
+---
+# Initial Instructions
+
+Before you are off and running with Compoze, there are a few one-time set up items to take care of. This allows the Compoze platform all the access rqeuired to full leverage all that Compoze has to offer.
+
+## Setup Steps
+
+### Step 1. Register a Route 53 Domain
+
+You will need to register a domain through AWS Route 53. Instructions can be found here: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html
+
+### Step 2. Create a Compoze IAM Role
+
+For detail instructions to create an IAM role, refer to: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html.
```

Comment:
@aleclarsen i think we can just give them a screenshot of what the role looks like, as well as the actual policy documents

Reply:
@larse514 do you mean give the IAM policy JSON? And a screenshot of what for the role?